GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1004,14 +1005,12 @@ void OnRevisionRead(GitRevision revision)
                     this.InvokeAsync(() => { ShowLoading(false); }).FileAndForget();
                 }
 
-                var isCurrentCheckout = revision.ObjectId.Equals(CurrentCheckout);
+                bool isCurrentCheckout = revision.ObjectId.Equals(CurrentCheckout);
 
-                if (isCurrentCheckout &&
-                    ShowUncommittedChangesIfPossible &&
-                    AppSettings.RevisionGraphShowWorkingDirChanges &&
```

Comment:
These checks have been moved to the other local function.
(Removing this setting would be OK for me since simpler git GUIs have similar artificial commits by default. Just my 2ct.)

Reply:
Artificial commits does not make sense in all forms, so at least `ShowUncommittedChangesIfPossible` should stay
In general I am in favor of reducing GE configurations, but it seems like each addition get someone confused, and removing old configurations would be worse.
These should be advanced settings though.